### PR TITLE
Add ability for a gamemaster to edit their table

### LINKED ISSUE #152: So that I can edit information for my table, as a game master, I want edit rights on my game_master's table
A game master should be able to see and edit their "gamemaster's" table.

### PR DESCRIPTION
**Pull request by Silbeg**
Also added Alex's position
Updated permissions on game master controller to open to hosts
Updated layout of game master and table for viewing and editing
Added some breadcrumbs

fixes #152

### CODE REVIEW

File: app/views/game_masters/edit.html.erb [hunk 1]
```diff
@@ -1,4 +1,48 @@
...
+</div>
+
+<% player = @game_master&.user_event&.user %>
+<%#
+table info
+table number
+session ?
+%>
+<h4>Table Information</h4>
+<div class="panel panel-padded" style="border: solid thin black;">
```
Comment (panpear) [inline app/views/game_masters/edit.html.erb:24]:
```suggestion
<div class="panel panel-padded thin-border">
```

File: app/views/game_masters/_form.html.erb [hunk 2]
```diff
@@ -12,59 +12,67 @@
   <% end %>
 
 
+  <h4>Online Game Information</h4>
+  <div class="panel panel-padded" style="border: solid thin black;">
```
Comment (panpear) [inline app/views/game_masters/_form.html.erb:16]:
```suggestion
  <div class="panel panel-padded thin-border">
```

File: app/views/game_masters/show.html.erb [hunk 3]
```diff
@@ -1,22 +1,79 @@
...
-  <%= @game_master.table_number %>
-</p>
+<% player = @game_master&.user_event&.user %>
+<%#
+table info
+table number
+session ?
+%>
+<h4>Table Information</h4>
+<div class="panel panel-padded" style="border: solid thin black;">
```
Comment (panpear) [inline app/views/game_masters/show.html.erb:27]:
```suggestion
<div class="panel panel-padded thin-border">
```

File: app/views/game_masters/show.html.erb [hunk 4]
```diff
@@ -1,22 +1,79 @@
...
+    <div class="col-md-6 col-sm-5"><%= @table.location %></div>
+  </div>
+  <div class="row">
+    <div class="col-md-2 col-sm-3"><strong>Table Information:</strong></div>
+    <div class="col-md-6 col-sm-5">Put stuff here</div>
+  </div>
+</div>
 
+<h4>Game Master's Information</h4>
+<div class="panel panel-padded" style="border: solid thin black;">
```
Comment (panpear) [inline app/views/game_masters/show.html.erb:39]:
```suggestion
<div class="panel panel-padded thin-border">
```

File: app/views/game_masters/show.html.erb [hunk 5]
```diff
@@ -1,22 +1,79 @@
...
+  <div class="row">
+    <div class="col-md-2 col-sm-3"><strong>Game Master:</strong></div>
+    <div class="col-md-6 col-sm-5"><%= player.formal_name %></div>
+  </div>
+  <div class="row">
+    <div class="col-md-2 col-sm-3"><strong>Email:</strong></div>
+    <div class="col-md-6 col-sm-5"><%= player.email %></div>
```
Comment (panpear) [inline app/views/game_masters/show.html.erb:46]: Should the game master be able to see the player's email? If it's to send them messages about the game, it'd probably be better to give them a mailing list for the game where they can't see the player's email directly. (Could we even make a mailing list per game? Is that a power GM's should even have?) If GMs need to communicate with players, would just having the game server do?

File: app/views/game_masters/show.html.erb [hunk 6]
```diff
@@ -1,22 +1,79 @@
...
+  </div>
+</div>
 
 
+<h4>Online Game Information</h4>
+<div class="panel panel-padded" style="border: solid thin black;">
```
Comment (panpear) [inline app/views/game_masters/show.html.erb:52]:
```suggestion
<div class="panel panel-padded thin-border">
```

File: app/views/game_masters/show.html.erb [hunk 7]
```diff
@@ -1,22 +1,79 @@
...
+  <div class="row">
+    <div class="col-md-2 col-sm-3"><strong>Chat Server:</strong></div>
```
Comment (panpear) [inline app/views/game_masters/show.html.erb:54]: What's the difference between a chat server and a chat room? Like the chat server is the overall discord, and the chat room is the specific channel for the game? If so, I'd call the other one 'Game Chat Room'

Comment (Silbeg) [reply app/views/game_masters/show.html.erb:54]: @panpear yes. 
Chat server will be the con's chat server.
The room is the specific room for the game.

File: app/views/game_masters/show.html.erb (hunk 5 above)
Comment (Silbeg) [reply app/views/game_masters/show.html.erb:46]: Probably not... this page is going to be updated with the next set of changes.

**Pull request merged**